docs/content/guides/developer/app-examples/reviews-rating.mdx

```diff
@@ -0,0 +1,451 @@
+
+### Review readers
+
+Review readers access reviews to make informed decisions on selecting services. Readers rate reviews by casting votes. The review readers' ratings are factored into the algorithm that rates the reviews, with the authors of the highest-rated reviews getting rewarded. Although it is not implemented as part of this guide, this example could be extended to award review readers a portion of the rewards for casting votes for reviews.
+
+### Moderators
+
+Moderators monitor content of the reviews and can delete any reviews that contain inappropriate content.
+
+The incentive mechanism for moderators is not implemented for this guide, but service owners can all pay into a pool that goes to moderators on a rolling basis. People can stake moderators to influence what portion of the reward each moderator gets, up to a limit (similar to how validators are staked on chain), and moderator decisions are decided by quorum of stake weight. This process installs incentives for moderators to perform their job well.
+
+## How reviews are scored
+
+The reviews are scored on chain using the following criteria:
+- Intrinsic score (IS): Length of review content.
+- Extrinsic score (ES): Number of votes a review receives.
+- Verification multiplier (VM): Reviews with PoE receive a multiplier to improve rating.
+
+```
+Total Score = (IS + ES) * VM
+```
+
+## Smart contracts
+
+There are several modules that create the backend logic for the example.
+
+### dashboard.move
+
+The `dashboard.move` module defines the `Dashboard` struct that groups services.
+
+```move
+/// Dashboard is a collection of services
+public struct Dashboard has key, store {
+    id: UID,
+    service_type: String
+}
+```
+
+The services are grouped by attributes, which can be cuisine type, geographical location, operating hours, Google Maps ID, and so on. To keep it basic, the example stores only `service_type` (for example, fast food, Chinese, Italian).
+
+```move
+/// Creates a new dashboard
+public fun create_dashboard(
+    service_type: String,
+    ctx: &mut TxContext,
+) {
+    let db = Dashboard {
+        id: object::new(ctx),
+        service_type
+    };
+    transfer::share_object(db);
+}
+
+/// Registers a service to a dashboard
+public fun register_service(db: &mut Dashboard, service_id: ID) {
+    df::add(&mut db.id, service_id, service_id);
+}
+```
+
+A `Dashboard` is a [shared object](../../../concepts/object-ownership/shared.mdx), so any service owner can register their service to a dashboard.
```

Comment:
@nonvis could you add a description here of why we need to group services into a `Dashboard`?